account/tests/test_client.py

```diff
@@ -0,0 +1,54 @@
+from account.models.profile import Profile
+from account.models.client import Client
+from .test_profile import CITY
+from datetime import datetime
+import pytest
+
+
+BIRTHDAY = datetime(2000, 1, 1)
+
+
+@pytest.fixture
+def save_clients(client, client2):
+    client.profile_id.user_id.save()
+    client.profile_id.save()
+    client.save()
+    client2.profile_id.user_id.save()
+    client2.profile_id.save()
+    client2.save()
```

Comment:
i think this needs to be in `conftest.py`

Reply:
@guybeckenstein Done